provider/resource_keycloak_openid_client_scope.go

```diff
@@ -35,6 +37,15 @@ func resourceKeycloakOpenidClientScope() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"include_in_token_scope": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"gui_order": {
+				Type:     schema.TypeString,
```

Comment:
Any reason for not using `schema.TypeInt` for this attribute?

Reply:
Keycloak REST API treats it as string type. Also, to remove the attribute, empty string ("") is used in admin console. That's why I use `schema.TypeString` here.

Reply:
I understand why you made that decision, but I think it would be better to change the type of this attribute to a `schema.TypeInt` anyways.  Unfortunately, there is a lot of areas for improvement for the Keycloak REST API, so I don't want to exactly model provider resources based on the types that the API uses.

We can convert the integer from the provider to a string to make the API call, and we can treat 0 as an empty string for the purpose of omitting the attribute to reset the order.

Reply:
All right. I'll change the code.